Intro/shared/src/commonMain/kotlin/io/realm/example/kmmsample/Calculator.kt

```diff
@@ -30,7 +31,7 @@ class Calculator private constructor() {
             return a + b
         }
 
-        fun listen(): Flow<List<Expression>> {
+        fun listen(): Flow<RealmResults<Expression>> {
```

Comment:
Any reason for this change? I assumed we used `List` to not depend too much on Realm specific things in the UI layer?